Description: exclude vendored directories
 Exclude the vendored directories from the workspace so the cargo
 build can succed.
Author: Julian Andres Klode <juliank@ubuntu.com>

--- lace-0.1.0.orig/Cargo.toml
+++ lace-0.1.0/Cargo.toml
@@ -11,7 +11,7 @@ members = [
     "tools/pewrap",
     "tools/xtask",
 ]
-
+exclude = [ "rust-vendor", "debian/merged-vendor" ]
 
 [profile.dev]
 panic = "abort"
